components/AnimationTiming/examples/AnimationTimingExample.m

```diff
@@ -73,14 +73,16 @@ - (void)applyAnimationToView:(UIView *)view
         view.transform = transform;
       }
       completion:^(BOOL finished) {
-        [UIView mdc_animateWithTimingFunction:timingFunction
-                                     duration:kAnimationTimeInterval
-                                        delay:kAnimationTimeDelay
-                                      options:0
-                                   animations:^{
-                                     view.transform = CGAffineTransformIdentity;
-                                   }
-                                   completion:nil];
+        if (finished) {
```

Comment:
This is a change in behavior, and I'm not confident that it's a desirable one. Consider reverting and marking `finished` as unused instead?